test/index.html

```diff
@@ -60,6 +60,12 @@ <h1>Hello, world!</h1>
 
             <p><a class="btn btn-primary btn-lg btn-success" href="#" role="button">Continue</a></p>
 
+            <article id="article">
```

Comment:
I think this might actually have been accidentally pulled in from the work I was doing on the a11y-text-wand plugin.

I'll make sure to clean this up before moving this PR out of the Draft state if we want to move forward with it.

Reply:
NP